### DISCUSSION TITLE
Add an extension to support hiding endpoints

### DISCUSSION THREAD

**Discussion by pardeep-kashyap** [scalar/scalar#1741] [Ideas]
Add functionality to exclude specific endpoints or private endpoints that we don't want to expose from being generated in the documentation

Discussion [link](https://discord.com/channels/1135330207960678410/1135330208627564597/1238458222562512928)

**Comment by hanspagel** [comment]
We built that yesterday! Should be released already. 😁

https://github.com/scalar/scalar/pull/1729

**Reply by pardeep-kashyap** [reply]
Great 🚀
